### DISCUSSION TITLE
Batch upload names with multiple Vouchers to a checklist

### DISCUSSION THREAD

**Discussion by Barkworth** [Symbiota/community-board#449] [Q&A]
I have downloaded the list of names not in a checklist I am creating. Some of the names are supported by multiple specimens. I am hoping someone knows (or would be intrigued enough to write) is an Excel expression for concatenating the catalog numbers of records vouchering the same taxon.  Write now, I use =concat(B16,", ",B17,", ",B18) 
[AJK Missing.xlsx](https://github.com/BioKIC/symbiota-docs/files/13995419/AJK.Missing.xlsx)
adding the cell numbers by hand - then copying and pasting as value the results of the expression into the "source" field of the first record for a given name  Using the link function in Symbiota is also time consuming. This seemed like a good approach but ...

**Comment by fbungartz** [comment]
Don't understand why you are not using the voucher tool batch linking?
Why the cumbersome down- and uploading of spreadsheets?
You can do all that already fully online...

**Comment by Barkworth** [comment]
Thank you. To be honest, I have forgotten what I was working on when I asked that question, but it is relevant to something else I am doing and I shall check out that voucher linking tool.

Mary
________________________________
